BUILD.md

```diff
@@ -50,6 +50,16 @@ Add the following to `onebusaway-android/gradle.properties`:
 
 ...where `XXXXXX` is your API key. Note that the suffix of `_oba` can be changed to configure API keys for other build flavors.
 
+### Configuration for OneSignal App ID
+
+If push arrivals notifications are active, you'll need to provide an App ID for [OneSignal](https://onesignal.com/).
+
+Add the following to `onebusaway-android/gradle.properties`:
```

Comment:
I see you have `ONESIGNAL_APP_ID="your_onesignal_app_id"` in gradle.properties in the project root. Which gradle.properties should this be set in, and can you clarify whether the key should be `ONESIGNAL_APP_ID` or `OneSignal_App_ID`?

Reply:
ONESIGNAL_APP_ID

Reply:
I have updated the `build.md` with the correct configurations